sdk/keyvault/Azure.Security.KeyVault.Keys/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
   "TagPrefix": "net/keyvault/Azure.Security.KeyVault.Keys",
-  "Tag": "net/keyvault/Azure.Security.KeyVault.Keys_a8557a0a67"
+  "Tag": "net/keyvault/Azure.Security.KeyVault.Keys_90068800ad"
```

Comment:
There is one test in Keys that depends on KeyVault.Secrets. Playback will fail unless re-recording with version `7.6-preview.2`